x-pack/plugin/esql/qa/testFixtures/src/main/resources/floats.csv-spec

```diff
@@ -592,3 +592,10 @@ s:double | emp_no:integer | salary:integer  | salary_change:double
 -1.0     | 10004          | 36174           | [-0.35, 1.13, 3.65, 13.48]
 -1.0     | 10005          | 63528           | [-2.14, 13.07]
 ;
+
+scalb
+row x = 3.0, y = 10 | eval z = scalb(x, y);
```

Comment:
To make this pass in mixed-cluster tests, you'll need a new capability (see EsqlCapabilities) and you'll have to add a `required_capability` to the test. You'll find a lot of examples of how to do it in CSV tests.